molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
I don't think this is true. A callback plugin only adds information to the ansible output, so it cannot really conflict with anything. 

In the case of my plugin, it adds lines that are easy to identify with a regex to the playbook summary. This summary is used by the updated idempotence function and if the information is not found, it falls back to the previous behaviour.

My point is that for a test tool, you do not want the user to the change the output. How often do you want to change the output of ``Test Infra``, ``Testtools``, or ``py.test``? 

I think this is necessary, because at the moment, the idempotence test says only ``PASSED`` or ``FAILED``, and in case of failure we do not know directly which tasks or handlers caused the failure. We have to replay a ``molecule converge`` to get the necessary information to fix the idempotence. That's not really a user friendly behaviour, and that's what I am trying to address.